### ISSUE TITLE
Make it possible to save as UTF-8

### ISSUE THREAD

**Issue by jberkel** [dulaccc/igenstrings#4]
Looks like Apple now recommends the use of UTF-8 for string files:

>It is recommended that you save strings files using the UTF-8 encoding, which is the default encoding for standard strings files.  Xcode automatically transcodes strings files from UTF-8 to UTF-16 when they’re copied into the product.

https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/LoadingResources/Strings/Strings.html

Maybe just expose the encoding parameter in the outer `save()` method while keeping the current default?

```python
def save(self, output_filename, encoding='utf16'):
   with io.open(output_filename, encoding=encoding, mode='w') as f:
```

**Comment by dulaccc** [comment]
Thanks for the heads-up! Nice they are changing their recommendation ^^
I will add new tests with that in mind, authorizing for both encoding to be supported

**Comment by dulaccc** [comment]
I bumped a new version with the fix `1.1.0`.
Thanks for bringing that to my attention.

Now the tool only support utf-8 encoding, which also plays nicely with `git` without any hack.

Cheers
